CHANGES/275.bugfix

```diff
@@ -0,0 +1,5 @@
+Update pulp_container deps to 2.3
+
+And some adjustments to use the newer
+pulpcore 3.10, and pulp_ansible 2.7ish that
+are needed for newer pulp_container
```

Comment:
copying comment from another pr:
> towncrier supports more than a single line in a changelog file, but I'm not sure we need to provide more than 1 line in most cases
> For this particular issue, since changelog is focused more towards users than devs, probably don't need extra info

This is a `.bugfix` but in the PR and JIRA issue I don't see a bug described